### PR TITLE
added Egyptian holidays

### PR DESCRIPTION
**Pull request by KommuSoft**
For a project, We use Egyptian public holidays, so I added these to the module.

It will add an additional requirement: `convertdate` since Egypt has Islamic holidays.

### CODE REVIEW

File: holidays.py [hunk 1]
```diff
@@ -2194,6 +2196,68 @@ class NL(Netherlands):
     pass
 
 
+class Egypt(HolidayBase):
+    SUNDAY = 1
```
Comment (coder6464) [inline holidays.py]: this variable is not used, what's its purpose?

File: holidays.py [hunk 2]
```diff
@@ -2194,6 +2196,68 @@ class NL(Netherlands):
...
+
+    def __init__(self, **kwargs):
+        # http://www.iamsterdam.com/en/plan-your-trip/practical-info/public-holidays
```
Comment (coder6464) [inline holidays.py]: can delete this comment, it is from the Netherlands section

File: holidays.py [hunk 3]
```diff
@@ -2194,6 +2196,68 @@ class NL(Netherlands):
...
+        self.country = "NL"
```
Comment (coder6464) [inline holidays.py]: change country code to "EG" like line 2257

File: holidays.py [hunk 4]
```diff
@@ -2194,6 +2196,68 @@ class NL(Netherlands):
...
+        HolidayBase.__init__(self, **kwargs)
+
+    def _populate(self, year):
+
+        # Christmas
+        self[date(year, 1, 7)] = "عيد الميلاد المجيد"
```
Comment (coder6464) [inline holidays.py]: would you consider writing the holiday names in ASCII English instead of Unicode Arabic?

File: holidays.py (hunk 1 above)
Comment (KommuSoft) [reply holidays.py]: It was copied and modified from the `Netherlands` class, but indeed the variable is not used.

**Comment by dr-prodigy** [comment]
Hi Kommusoft, thanks for your contribution. Please:
- fix Travis CI issues
- provide complete tests for all your new code
in order to get it pulled.
Thank you!

**Comment by KommuSoft** [comment]
Hi,

Ik will try to fix it by tuesday evening if that is all right?

**Comment by dr-prodigy** [comment]
Hi,
      of course, do it when you can!
      Thank you
      
      On 01/29/2018 01:16 AM, Willem Van Onsem wrote:
    
    
      Hi,
      Ik will try to fix it by tuesday evening if that is all right?
      —
        You are receiving this because you commented.
        Reply to this email directly, view it on GitHub, or mute the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/dr-prodigy/python-holidays","title":"dr-prodigy/python-holidays","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/dr-prodigy/python-holidays"}},"updates":{"snippets":[{"icon":"PERSON","message":"@KommuSoft in #96: Hi,\r\n\r\nIk will try to fix it by tuesday evening if that is all right?"}],"action":{"name":"View Pull Request","url":"https://github.com/dr-prodigy/python-holidays/pull/96#issuecomment-361110010"}}}

**Comment by dr-prodigy** [comment]
Hi, I close: please submit a new PR when fixes are released.
Thank you see you soon!
dr-p

**Pull request closed without merging**